argo-cd-apps/environments/production/kustomization.yaml

```diff
@@ -9,7 +9,8 @@ resources:
   - ../../dependencies/pipelines-as-code  
   - ../../dependencies/tekton-config
   - ../../dependencies/tekton-operator
-  - ../../dependencies/tekton-results  
```

Comment:
There are some trailing whitespaces on some of the lines above. Could you remove them as well?

Reply:
Same in staging

Reply:
We need to setup a CI for this repository, add lint and other tests

Reply:
done